From a365684bf086742763b098feddaf5525bc9581c0 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 07 May 2012 08:33:44 +0000
Subject: configure.ac: more AS_IF

It looks like JH_CHECK_XML_CATALOG has some prerequisite as well

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

https://bugzilla.gnome.org/show_bug.cgi?id=674483
---
diff --git a/configure.ac b/configure.ac
index 37daa54..84a48bc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2591,7 +2591,7 @@ AC_ARG_ENABLE(man,
                               [regenerate man pages from Docbook [default=no]])],enable_man=yes,
               enable_man=no)
 
-if test "${enable_man}" != no; then
+AS_IF([test "${enable_man}" != no], [
 dnl
 dnl Check for xsltproc
 dnl
@@ -2605,7 +2605,7 @@ AC_PATH_PROG([XSLTPROC], [xsltproc])
      [DocBook XML DTD V4.1.2],,enable_man=no)
   JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl],
      [DocBook XSL Stylesheets],,enable_man=no)
-fi
+])
 
 AM_CONDITIONAL(ENABLE_MAN, test x$enable_man != xno)
 
--
cgit v0.9.0.2
